.travis.yml

```diff
@@ -29,4 +31,4 @@ install:
 script:
     - black --check inverse_covariance
     - black --check examples
-    - python -m pytest --showlocals --pyargs
+    - pytihon -m pytest --showlocals --pyargis
```

Comment:
@mnarayan is it possible this no longer throws test errors because its no longer running the tests?  `pytihon` looks like a typo

Reply:
Yeah I missed the typos. They are fixed now. But we still have test errors due to the bugs reported in #126 at the very least.